includes/class-give-donor.php

```diff
@@ -790,6 +790,29 @@ public function get_notes_count() {
 	}
 
 	/**
+	 * Get the total donation amount.
+	 *
+	 * @since 1.8.17
+	 *
+	 * @param array $args Pass any additional data.
+	 *
+	 * @return string|float
+	 */
+	public function get_total_donation_amount( $args = array() ) {
```

Comment:
@DevinWalker Please review new function with filter to get total donation amount